File: backend/aums/src/main/java/com/accolite/aums/models/Course.java
/**
 * 
 */
package com.accolite.aums.models;


import java.sql.Timestamp;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * @author darshan
 *
 */
@Getter @Setter @ToString
public class Course {

	private int courseId;
	private int creatorId;
	private String courseName;
	private String courseDesc;
    private String courseLocation ;
    private String courseSkill;
    private String coursePrerequisites;
    private String courseMonth;
    private String courseYear;
    private Timestamp modifiedAt;
    
}

File: frontend/AUMS/src/app/models/Course.ts
import { Instructor } from './Instructor';
import { Timestamp } from 'rxjs/internal/operators/timestamp';

export class Course {
    courseId: number;
    courseName: string;
    courseSkill: string;
    courseLocation: string;
    courseMonth: string;
    courseYear: string;
    coursePrerequisites: string;
    instructor: Instructor[];
    modifiedAt: string;
}